python/Scripts/baketextures.py

```diff
@@ -10,6 +10,16 @@
 from MaterialX import PyMaterialXRender as mx_render
 from MaterialX import PyMaterialXRenderGlsl as mx_render_glsl
 
+def tobool(v):
```

Comment:
Ah, I see that there isn't a universally-available `strtobool` method in Python right now:

https://stackoverflow.com/questions/42248342/yes-no-prompt-in-python3-using-strtobool

Could we instead add a `stringToBoolean` method to `datatypes.py` in MaterialX Python, allowing our example scripts to access `mx::stringToBoolean` in a consistent way?

Reply:
With a `stringToBoolean` method available, we may then be able to directly reference this as the `type` of the argument in our parser, as described in this post:

https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse

Reply:
Good idea. 

I've done this but I needed to add `argpase` as a dependency to throw a proper exception. The "simpler"
options which just return `false` if not parsed are not correct if the default value is not `false` (which happens in the current case). Thus we throw an exception as shown in the sample code when a bool cannot be parsed. I removed the 'y' and 'n' options as most just parse 't', 'f', '0', '1', 'True', 'False', True, False.